src/elm/Help/Help.elm

```diff
@@ -177,7 +180,7 @@ row content copy =
             Just copyMsg ->
                 copyButton
                     [ Util.testAttribute "help-copy"
-                    , attribute "aria-label" "view cli command for this page"
+                    , attribute "aria-label" <| "copy " ++ content ++ " to clipboard"
```

Comment:
added better aria label to copy button to give screen readers information as to what the button is doing AND copying